testing.rst

```diff
@@ -51,79 +50,111 @@ Symfony application.
     Use the ``--coverage-*`` command options to generate code coverage reports.
     Read the PHPUnit manual to learn more about `code coverage analysis`_.
 
-.. index::
-   single: Tests; Unit tests
+Types of Tests
+--------------
+
+`Unit Tests`_
```

Comment:
I really like that we define these types of tests. But since many definitions already exist, I suggest that we add a disclaimer with the following principle: 

"This is how we define the test type in the documentation, It is just because we need to know what we are referring to. You may have learnt something else, that is not necessarily wrong".